packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -659,6 +659,10 @@ describe('Select', () => {
             const { element, connect, disconnect } = await setup500Options();
             await connect();
             await clickAndWaitForOpen(element);
+            element.listbox.style.setProperty(
+                '--ni-private-listbox-num-visible-options',
+                '10000'
+            );
```

Comment:
Make sure we're exercising the viewport limit rather than the visible option limit.